web/session.go

```diff
@@ -0,0 +1,91 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package web
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"time"
+
+	"github.com/cloudspannerecosystem/harbourbridge/conversion"
+)
+
+// Metadata of session file, which contains app.conv in
+// JSON format.
+type Session struct {
+	Driver    string    `json:"driver"`
+	FilePath  string    `json:"path"`
+	FileName  string    `json:"fileName"`
+	CreatedAt time.Time `json:"createdAt"`
+}
+
+func getSession(w http.ResponseWriter, r *http.Request) {
```

Comment:
About half of this code is the same as WriteSessionFile (conversion.go).
Can we call WriteSessionsFile here?

Reply:
updated function as mentioned